.github/ISSUE_TEMPLATE/work-item.yml

```
@@ -1,6 +1,6 @@
-name: "Work Item Issue Template"
-description: "Template for creating new work item issues"
-title: "[Work_Item]"
+name: "Work Item"
+description: "Template for creating new Work Items"
+title: "[WI]"
 labels: 
   - "work item"
```

Would like to make sure I'm always reviewing things in the `needs backlog review` bucket with the Maintainers, including new WIs.

```suggestion
  - "work item", "needs backlog review"
```